lms/static/js/student_account/views/LoginView.js

```diff
@@ -29,13 +30,20 @@
                 this.errorMessage = data.thirdPartyAuth.errorMessage || '';
                 this.platformName = data.platformName;
                 this.resetModel = data.resetModel;
+                this.supportURL = data.supportURL;
 
                 this.listenTo( this.model, 'sync', this.saveSuccess );
                 this.listenTo( this.resetModel, 'sync', this.resetEmail );
             },
 
             render: function( html ) {
                 var fields = html || '';
+                var message = 'We have sent an email message with password reset instructions to the email ' +
+                'address you provided.  If you do not receive this message, {anchorStart}contact edX support{anchorEnd}.';
```

Comment:
We must not have "edX" in our text strings. It is a little tricky to get the configured product name down to the JS but that may already have been done in student_account. If not, you could reword it to something like ``contact technical support``. I recommend looking at other places that do this.